recipes/pkutils/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "pkutils" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 948fe5acb8fd9f315c508756a6420d126400c470e7111408521bb0c7d409c280
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
```

```suggestion
  host:
```
sorry, my bad, `host` was correct...